provider/github/settings.ts

```diff
@@ -0,0 +1,6 @@
+export type Settings = {
+    /** The accessToken to use to talk to the GitHub API. */
+    accessToken: string
+    /** Optional base URL of a GitHub enterprise instance, when not using GitHub.com. */
```

Comment:
its now called github enterprise server, and github ae is another thing that's not "GHE" but that has a different base URL.

check out how their cli documents this param and try to follow that for consistency: https://cli.github.com/manual/gh_auth_login

Reply:
Hmm their CLI seems to be not flexible enough, it only accepts a `hostname`, but doesn't mandate the protocol nor port. 
Unless they have some secret agreement that you can never run github on a port other than 443, that feels incorrect to me.

Unless you feel strongly about this, I will stick to using URLs here instead of the hostname for now, as that seems most compatible.
I will update the wording in the readme though.